build.gradle.kts

```diff
@@ -59,10 +59,14 @@ tasks {
     }
 
     shadowJar {
+        exclude("/prebuilt/linux/aapt")
+        exclude("/prebuilt/linux/aapt_64")
+        exclude("/prebuilt/macosx/aapt_64")
+        exclude("/prebuilt/windows/aapt.exe")
+        exclude("/prebuilt/windows/aapt_64.exe")
```

Comment:
Can't a wildcard be used to reduce the amount of exclude statements?

Reply:
I tried my best but it's not possible.
If I use a wildcard, it will always match aapt2.
Example: `exclude("/prebuilt/**/aapt*")` matches `/prebuilt/linux/aapt2`, `/prebuilt/linux/aapt2_64`

[Apache Ant style wildcard is supported here](https://gradleup.com/shadow/configuration/filtering/), but it has only three wildcards, `?`, `*`, and `**`
If I put `aapt*`, it also matches `aapt2*` always.
So there is no way to avoid including aapt2 binaries other than to list the five aapt1.

Reply:
What about something like this?

`
exclude("/prebuilt/*/aapt")
exclude("/prebuilt/*/aapt_*")
exclude("/prebuilt/*/aapt.*")
`